### PR TITLE
<wip> microcard feeback

### PR DESCRIPTION
**Pull request by cdaringe**
# discussion

PR staged just for discussion purposes

### CODE REVIEW

**Comment by cdaringe** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://d4bf6ada-cd5b-4e19-b981-e53689340d47.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/Tripwire/octagon/branch/microcard/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

**Comment by cdaringe** [comment]
nice!  this is on the money 🤑 

- would `<Card micro />` `<Card jumbo />` be achievable vs two separate components?
- prefer direct imports vs imports from suir directly

anyway, this is exactly what i was hoping for!

**Comment by eddier** [comment]
Since the components have unique fragments it might be better to keep them separated rather than rather than the unified "Card" parent component.

Can you explain what you mean or use an example to illustrate point 2 to make sure we're talking about the same thing?

**Comment by cdaringe** [comment]
> Since the components have unique fragments it might be better to keep them separated rather than rather than the unified "Card" parent component.

alrighty!

> Can you explain what you mean or use an example to illustrate point 2 to make sure we're talking about the same thing?

yea, that couldnt have been more unclear.
`import Blah from 'suir/comps/blah'` 👍 
vs
`import { Blah } from 'suir'` 👎

**Comment by eddier** [comment]
I'll update to reflect that.

**Comment by eddier** [comment]
@cdaringe that last push should deal with the issue you mentioned.  If we're happy with this approach I'll polish it up and this chunk should be ready for the first merge soon.

**Comment by jhegg** [comment]
I'm liking the direction this is going! I like that this has concepts like keeping the close button a separate component, because for example I may not always want a close button on the large card.

**Comment by eddier** [comment]
Yeah definitely - you can see a more developed example of this being implemented on the gitlab repo you were just added to @jhegg

File: src/styles/components/micro-card.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  width: 300px;
+  font-size: var(--small);
+
+  min-height: 86px;
+  position: relative;
+  & .text-uppercase {
+      text-transform: uppercase;
+  }
+  &.bordered {
+    /* border: 2px solid color(var(--yellow)); */
```
Comment (jhegg) [inline src/styles/components/micro-card.css]: Should this be removed?

**Comment by jhegg** [comment]
Cool, thanks. :smile:

File: src/styles/components/micro-card.css (hunk 1 above)
Comment (eddier) [inline src/styles/components/micro-card.css]: Yes - there is some clean up that needs to happen on these.  The components were originally designed as a 'whole'.  For example a microcard had a hard coded set of fields/layout that was expected, now the deconstruction of the component into smaller fragments has left some hangover in the CSS.  That said the CSS is filtered out on postprocessing to keep unused code out.

**Comment by cdaringe** [comment]
@eddier reopen or create new when ready!

**Pull request closed without merging**